script.js

```diff
@@ -1,70 +1,101 @@
 class Book {
-    constructor(title, author) {
-        this.title = title;
-        this.author = author;
-    }
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
```

Comment:
- [ ]  Your project lacks the time setting just below the navbar. For setting the time you were supposed to use the luxon library. However, I have noticed that the link is not working on the project requirements but we apologize for the inconvenience. Please refer to this [article](https://moment.github.io/luxon/#/) on how to set it up as the broken link gets updated. Unfortunately, this is a hard requirement for the project and needs to be implemented.